packages/stats/src/collector.ts

```diff
@@ -0,0 +1,109 @@
+import * as Parser from '@tracespace/parser'
+import {DrillUsage, DrillStats} from './types'
+
+/**
+ * Method to collect drill stats of parsed drill files.
+ *
+ * @example
+ * ```ts
+ * import {createParser} from '@tracespace/parser'
+ * import {collectDrillStats} from '@tracespace/stats'
+ *
+ * const parser = createParser()
+ *
+ * parser.feed(...)
+ *
+ * const tree = parser.results()
+ * const stats = collectDrillStats([tree])
+ * ```
+ *
+ * @category Stats
+ */
+export function collectDrillStats(trees: Parser.Root[]): DrillStats {
+  let toolsMap = new Map<string, Parser.ToolShape>()
+  let drillsPerTool = new Map<string, number>()
+  let routesPerTool = new Map<string, number>()
+  let minDrillSize: number = null
+  let maxDrillSize: number = null
+
+  for (const tree of trees) {
+    if (tree.filetype !== Parser.DRILL) {
+      continue
+    }
+
+    let currentTool = ''
```

Comment:
I would like to avoid using empty strings as "falsey" values. Prefer `null` for this purpose